Taskfile.yaml

```diff
@@ -0,0 +1,39 @@
+version: '3'
+
+
+dotenv: [".env", '{{.ENV}}/.env.']
+
+# NOTE: Task doesn't allow to override environment variables. Thus, when an
+# environment variable is referenced in a taskfile, it is because we expect it
+# to be defined by the environment where Task is being invoked, in an `.env`
+# file, or in a task's `env:` attribute.
+
+
+# if a task is referenced multiple times, only run it once
+run: once
+
+tasks:
+  kind-create:
+    cmds:
+      - kind create cluster --config ./.github/kind.yaml
```

Comment:
Should we add a name flag here instead of using the default?

Reply:
Not sure, dow we? usually i just use the name kind, but sure.